syntax = "proto3";

package avant;

option go_package = "/proto_res";

enum ProtoCmd
{
    PROTO_CMD_CS_REQ_EXAMPLE = 0;
    PROTO_CMD_CS_RES_EXAMPLE = 1;
    PROTO_CMD_TUNNEL_MAIN2WORKER_NEW_CLIENT = 2;
    PROTO_CMD_TUNNEL_PACKAGE = 3;
    PROTO_CMD_TUNNEL_CLIENT_FORWARD_MESSAGE = 4;
    PROTO_CMD_TUNNEL_WEBSOCKET_BROADCAST = 5;
    PROTO_CMD_TUNNEL_WORKER2OTHER_EVENT_NEW_CLIENT_CONNECTION = 6;
    PROTO_CMD_TUNNEL_OTHER2WORKER_TEST = 7;
    PROTO_CMD_LUA_TEST = 8;
    PROTO_CMD_IPC_STREAM_AUTH_HANDSHAKE = 9;
}
